### DISCUSSION TITLE
UTM Source showing as Direct

### DISCUSSION THREAD

**Discussion by sklink** [capawesome-team/capacitor-firebase#690] [Q&A]
I've added @capacitor-firebase/analytics. Traffic is showing in Google Analytics from the Measurement ID I've used to initialize. Initialization is happening through `import { initializeApp } from "firebase/app";`

However, adding `utm_source` as a querystring parameter when landing shows "Direct" as the source. We're tracking utm_source accurately by sending it to the DB, so we know it's being set. Seems as though it's not getting picked it up through the `@capacitor-firebase/analytics` plugin. 

I'm assuming this is something in my setup, as I haven't seen anyone else with this issue.

- "firebase" version is 10.8.1
- "@capacitor-firebase/analytics" version is 5.4.1

Any suggestions? Unsure where to dig for further investigation.

**Comment by robingenz** [comment]
I haven't tested this yet, but maybe someone from the community can help you.
